External/Plugins/CodeRefactor/Commands/OrganizeImports.cs

```diff
@@ -226,7 +225,24 @@ class ImportsComparerType : IComparer<MemberModel>
     {
         public Int32 Compare(MemberModel item1, MemberModel item2)
```

Comment:
I'd like that, unfortunately I have very little idea about C# in general and structure of FD in particular. (This being my first use of C#, actually). When I asked Mika how can I use the compare function from one plug-in in the other plug-in, he advised me to copy&paste it to avoid creating references between plug-ins...